src/test/resources/messages/krbLogin-request.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<methodCall>
+  <methodName>krbLogin</methodName>
+  <params>
+    <param>
+      <value>
+        <string>VGhlIHF1aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4KVGhlIHF1aWNrIGJy
+b3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4KVGhlIHF1aWNrIGJyb3duIGZveCBqdW1w
+cyBvdmVyIHRoZSBsYXp5IGRvZy4KVGhlIHF1aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBs
+YXp5IGRvZy4KVGhlIHF1aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4KVGhl
+IHF1aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4KVGhlIHF1aWNrIGJyb3du
+IGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4KVGhlIHF1aWNrIGJyb3duIGZveCBqdW1wcyBv
+dmVyIHRoZSBsYXp5IGRvZy4KVGhlIHF1aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5
+IGRvZy4KVGhlIHF1aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4KVGhlIHF1
+aWNrIGJyb3duIGZveCBqdW1wcyBvdmVyIHRoZSBsYXp5IGRvZy4=
+</string>
```

Comment:
Can you try to remove the last \n before </string>? It seems STAX thinks there are two CHAR events. With the first event, it read the whole string until =; with the second, it reads a "" and override the prev value.

Reply:
This changes things a bit. Now we seem to get the portion of the string after the last `\n` instead of an empty string in some cases. In any case, it looks like the final `String` value is just the last `CHAR` event that happens instead of the entire text between the `<string></string>` tags, if what you say is true.

However, it looks like you have found that my tests are wrong. The expected string must end in a newline according to the spec (and this is also what the server sends), so the XML files are correct. I will update the expected test strings to end in a newline (by the way, the newline characters may vary depending on which operating system the client and server are running), but I stuck to UNIX in both cases for simplicity.